TreeDB/cmd/treedb_rag_benchmark/minima.go

```
@@ -0,0 +1,1476 @@
+	if err := validateMinimaScalarNamespaces(manifest); err != nil {
+		return err
+	}
+	if err := validateMinimaTimedPlan(manifest); err != nil {
+		return err
+	}
+	if err := validateMinimaConcurrentMutationPlans(manifest); err != nil {
+		return err
+	}
+	stateHash, err := minimaExpectedStateHash(manifest)
+	if err != nil || manifest.ExpectedStateSHA256 != stateHash {
+		return fmt.Errorf("minima manifest: post-operation state hash mismatch: %v", err)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)